### PR TITLE
optimizations

### LINKED ISSUE #40: Client ID should be trimmed of spaces etc

### PR DESCRIPTION
**Pull request by thde**
resolves #40

### CODE REVIEW

**Comment by lroellin** [comment]
Can you mention what issues this closes, so they close automatically?

 They probably have to be in the description, not in the comments.

File: src/main/kotlin/ch/hsr/dsl/dwrtc/util/TomP2P.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ch.hsr.dsl.dwrtc.util
```
Comment (lroellin) [inline src/main/kotlin/ch/hsr/dsl/dwrtc/util/TomP2P.kt]: Can you add the latest ones here?

**Comment by lroellin** [comment]
The rest LGTM.

File: src/main/kotlin/ch/hsr/dsl/dwrtc/util/TomP2P.kt (hunk 1 above)
Comment (lroellin) [reply src/main/kotlin/ch/hsr/dsl/dwrtc/util/TomP2P.kt]: On the other hand, we agreed to waiting until we integrate it into the challenge project, so maybe you'll do this afterwards

Comment (thde) [reply src/main/kotlin/ch/hsr/dsl/dwrtc/util/TomP2P.kt]: I'd do this in a later commit.

**Pull request merged**